index.js

```diff
@@ -40,12 +40,39 @@ class ServerlessPythonRequirements {
   };
 
   /**
+   * parse requirements.txt into .requirements.txt, leaving out #no-deploy lines
+   * @return true
+   */
+  parseRequirements() {
+    if (!fse.existsSync(path.join(this.serverless.config.servicePath,
+                                  'requirements.txt'))) {
+      return BbPromise.resolve();
```

Comment:
Should probably `BbPromise.reject()` since in this scenario the `pip install` command will fail.